Entity/AbstractCrudEntity.php

```diff
@@ -20,7 +23,7 @@
  *
  * @author Sergey Chernecov <sergey.chernecov@intexsys.lv>
  */
-abstract class AbstractCrudEntity implements CrudEntityInterface
+abstract class AbstractCrudEntity implements CrudEntityInterface, EmbeddedInterface, TransactionalInterface, TimestampableInterface
```

Comment:
@RuslanZavacky I've moved these interfaces from the CrudEntityInterface to here but don't really think they ideally belong here. I've done this to preserve backwards compatibility but think we could use a new abstract entity rather than this one to implement all these interfaces. I do not think general CRUD needs transactions, timestamps, etc...